TODO.md

```diff
@@ -20,6 +20,7 @@
   - [ ] os.kill
   - [ ] os.killpg
   - [ ] os.plock
+  - [ ] os.removedirs
   - [ ] execfile
```

Comment:
Why?

Reply:
`execfile` is just more python....

Reply:
It is for executing other files; that could be dangerous.

Reply:
I know, but wouldn't all the imports will be executed in the main module level of the original process?

Reply:
> I know, but wouldn't all the imports will be executed in the main module level of the original process?

I guess...
I removed it anyway. (someone could probably use something else to do the same thing - `eval(open("file.py").read())`?)

Reply:
Sure... Using `exec` makes more sense, but you're right (if it's actually in the main module level of the original process)

Reply:
Btw, `execfile` was removed in python3 and now you suppose to use `exec` with `open` anyways